src/Calibr/EhApiClient/Client.php

```diff
@@ -116,9 +118,20 @@ private function _request($method, $url, $body = null, $options = []) {
     }
     $url = $this->_prepareUrl($url);
     $url = $this->_baseUrl . $url;
-    $res = $this->_callClient($method, $url, $options);
-    $this->_checkResponseError($res);
-    return $this->_parseResponse($res);
+    while ($this->currentTry <= $this->maxNumberOfTries) {
+      $this->currentTry++;
```

Comment:
Why saving number of tries in the client, maybe it is better to just define a local variable in the function?